### PR TITLE
Update samsung backend settings 

### PR DESCRIPTION
**Pull request by AhmedTElthakeb**

### CODE REVIEW

File: mobile_back_samsung/cpp/sbe_backend/jni/include/sbe_config.hpp [hunk 1]
```diff
@@ -130,13 +129,14 @@ benchmark_setting {
...
-    value: "200"
+    value: "300"
   }
   custom_setting {
     id: "lazy_mode"
-    value: "true"
+    value: "false"
   }
-  src: "https://github.com/mlcommons/mobile_models/raw/main/v2_0/Samsung/ic_single_fence.nnc"
+  src: "/sdcard/MLPerf_sideload/ic_single.nnc"
```
Comment (anhappdev) [inline mobile_back_samsung/cpp/sbe_backend/jni/include/sbe_config.hpp:138]: Why is the `src` updated to use local file?

File: mobile_back_samsung/samsung_backend.mk [hunk 2]
```diff
@@ -16,4 +16,39 @@
...
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libvndksupport.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libbase.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libcgrouprc.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libhidlbase.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libgpu_boost_vendor.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libprocessgroup.so \
+    ${MOBILE_BACK_SAMSUNG_LIB_ROOT}/libvendor_samsung_slsi_hardware_gpu_boost.so
+  # main binaries for Samsung backend are prebuilt
+  backend_samsung_android_target=//android/commonlibs:commonlibs
+  backend_samsung_filename=libsamsungbackend
```
Comment (anhappdev) [inline mobile_back_samsung/samsung_backend.mk:53]: All of this was removed in `android-v2` branch because it's only used for Flutter build.

**Review by anhappdev — CHANGES_REQUESTED** [review]

**Review by anhappdev — APPROVED** [review]

**Comment by anhappdev** [comment]
@AhmedTElthakeb APK built from this branch with `*.so` files from `mobile_back_samsung/mobile_back_samsung/samsung/lib_060622`:
[2022-06-13_PR351_mlperf_app.apk](https://drive.google.com/file/d/1PgjbpTAuAcawWde-hOpXfTL51AborDbL/view?usp=sharing)

**Comment by AhmedTElthakeb** [comment]
> @AhmedTElthakeb APK built from this branch with `*.so` files from `mobile_back_samsung/mobile_back_samsung/samsung/lib_060622`: [2022-06-13_PR351_mlperf_app.apk](https://drive.google.com/file/d/1PgjbpTAuAcawWde-hOpXfTL51AborDbL/view?usp=sharing)

Tested and seems to work fine.

**Comment by anhappdev** [comment]
@AhmedTElthakeb Please resolve the conflict to merge the changes into `android-v2`.

**Pull request merged**